src/Storefront/Resources/snippet/de_DE/storefront.de-DE.json

```diff
@@ -1,480 +1,480 @@
-        "reviewExistsTeaserText": "Sie können hier Ihre bestehende Bewertung anpassen.",
-        "reviewExistsTeaserButton": "Bewertung bearbeiten",
-        "reviewLanguageFilterLabel": "Bewertungen nur in aktueller Sprache anzeigen.",
-        "reviewSortLabel" : "Sortiert nach",
-        "reviewSortTopRatedLabel" : "Am besten bewertet",
-        "reviewSortNewLabel" : "Neueste Bewertung",
-        "reviewCountAfter": "{1} Bewertung|]1,Inf[ Bewertungen",
-        "reviewCountBefore": "von",
-        "reviewListEmpty": "Keine Bewertungen gefunden. Seien Sie der Erste und teilen Sie Ihre Gedanken anderen mit.",
-        "reviewLoginHeader": "Anmelden",
-        "reviewLoginDescription": "Bewertungen können nur von angemeldeten Benutzern abgegeben werden. Bitte geben Sie Ihre Login-Daten ein, um sich einzuloggen, oder erstellen Sie einen neuen Account.",
-        "reviewRegisterLink": "Neuer Kunde?",
-        "reviewLoginSignupLink" : "Klicken Sie hier, um sich anzumelden.",
-        "reviewLoginCancelButton": "Abbrechen",
-        "reviewAvgRate": "von",
-        "reviewAvgRateElements": "Sternen"
-    },
-    "footer": {
-        "serviceHotlineHeadline": "Service-Hotline",
-        "shopServiceHeadline": "Shop-Service",
-        "informationHeadline": "Informationen",
-        "newsletterHeadline": "Newsletter",
-        "serviceHotline": "Unterstützung und Beratung unter: <a href=\"tel:+49180000000\">0180 - 000000</a> Mo-Fr, 09:00 - 17:00 Uhr",
-        "serviceContactLink": "Oder über unser <a data-toggle=\"modal\" data-url=\"%url%\" href=\"%url%\" title=\"Kontaktformular\">Kontaktformular</a>.",
-        "newsletterInput": "Ihre E-Mail-Adresse",
-        "newsletter": "Abonnieren Sie jetzt einfach unseren regelmäßig erscheinenden Newsletter und Sie werden stets unter den Ersten sein, über neue Produkte und Angebote informiert werden.",
-        "includeVat": "* Alle Preise inkl. gesetzl. Mehrwertsteuer zzgl. <a href=\"#\">Versandkosten</a> und ggf. Nachnahmegebühren, wenn nicht anders beschrieben",
-        "excludeVat": "* Alle Preise verstehen sich zzgl. Mehrwertsteuer und <a href=\"#\">Versandkosten</a> und ggf. Nachnahmegebühren, wenn nicht anders beschrieben",
-        "copyrightInfo": "Realisiert mit Shopware"
-    },
-    "cookie": {
-        "message": "Diese Website verwendet Cookies, um Ihnen die bestmögliche Funktionalität bieten zu können. <a data-toggle=\"modal\" data-url=\"%url%\" href=\"%url%\" title=\"Mehr Informationen\">Mehr Informationen</a>.",
-        "deny": "Ablehnen",
-        "configure": "Konfigurieren",
-        "configuration": "Konfiguration",
-        "save": "Speichern",
-        "headline": "Cookie Einstellungen",
-        "groupRequired": "Technisch erforderlich",
-        "groupRequiredDescription": "Diese Cookies sind für die Grundfunktionen des Shops notwendig.",
-        "groupRequiredSession": "Session",
-        "groupRequiredTimezone": "Zeitzone"
-    }
```

Comment:
Please don't change indents globally.

Reply:
Anyone has to do it, related https://github.com/shopware/platform/blob/6.1/.editorconfig#L20

Reply:
But the Indentlength is 2 not 4...?

Reply:
editorconfig says 2 for json 🤔

Reply:
Yes, and you did 4 :D

Reply:
No, I changed it from 4 to 2.

Reply:
Friday Morning. I'm sorry for my stupidity :D You're totally right.

But i hope there aren't other changes within. Would be horrible to review them

Reply:
There are some fixes in de-DE. You can check them with "w=1"-Parameter, to hide whitespace-changes :-)
https://github.com/shopware/platform/pull/479/files?w=1#diff-6f3c3eea16c822f9ca3a0b5a38c514aa

Reply:
Didn't know that. Nice to know. Thx!